website/source/docs/providers/aws/r/api_gateway_method_settings.html.markdown

```diff
@@ -0,0 +1,93 @@
+---
+layout: "aws"
+page_title: "AWS: aws_api_gateway_method_settings"
+sidebar_current: "docs-aws-resource-api-gateway-method-settings"
+description: |-
+  Provides an API Gateway Method Settings, e.g. logging or monitoring.
+---
+
+# aws\_api\_gateway\_method\_settings
+
+Provides an API Gateway Method Settings, e.g. logging or monitoring.
+
+## Example Usage
+
+```hcl
+resource "aws_api_gateway_method_settings" "s" {
+  rest_api_id = "${aws_api_gateway_rest_api.test.id}"
+  stage_name  = "${aws_api_gateway_stage.test.stage_name}"
+  method_path = "${aws_api_gateway_resource.test.path_part}/${aws_api_gateway_method.test.http_method}"
```

Comment:
Thanks for questioning this - I was actually playing with that idea too! 😄 

I'm not completely opposed to 2 fields (e.g. `path_part` & `method`).

The API allows you to define things like `*/*` or `*/GET` and I'm just little bit worried that they might later introduce (or even already support) some kind of special magic syntax which doesn't involve `/` and we'd prevent the user from specifying such paths. 🤔